File: Beginner/1037_Interval/.hs
import Text.Printf

main :: IO ()
main = do
  n <- readLn :: IO Float
  if n >= 0 && n <= 25.0000
    then printf "Intervalo [0,25]\n"
    else if n >= 25.00001 && n <= 50.0000000
           then printf "Intervalo (25,50]\n"
           else if n >= 50.00001 && n <= 75.0000000
                  then printf "Intervalo (50,75]\n"
                  else if n >= 75.00001 && n <= 100.0000000
                         then printf "Intervalo (75,100]\n"
                         else printf "Fora de intervalo\n"

File: Beginner/1037_Interval/.go
package main

import (
	"fmt"
)

func main() {
	var n float64


	fmt.Scan(&n)

	if(n < 0 || n > 100){
	 fmt.Printf("Fora de intervalo\n");
	   }else{
		if(n >= 0 && n <= 25){
		 fmt.Printf("Intervalo [0,25]\n");
		}else if(n > 25 && n <= 50){
		 fmt.Printf("Intervalo (25,50]\n");
		}else if(n > 50 && n <= 75){
		 fmt.Printf("Intervalo (50,75]\n");
		}else{
		 fmt.Printf("Intervalo (75,100]\n");
		}
	   }




}
